docs/languages/cpp.md

```diff
@@ -21,95 +21,112 @@ C/C++ support for Visual Studio Code is provided by a [Microsoft C/C++ extension
+   1. In the Windows search bar, type **Settings** to open your Windows Settings.
+   1. Search for **Edit environment variables for your account**.
+   1. In your **User variables**, select the `Path` variable and then select **Edit**.
+   1. Select **New** and add the MinGW-w64 destination folder you recorded during the installation process to the list. If you selected the default installation steps, the path is: `C:\msys64\mingw64\bin`.
+   1. Select **OK** to save the updated PATH. For the new `PATH` to be available, reopen your console windows.
+1. Check that your MinGW-w64 tools are correctly installed and available, open a **new** Command Prompt and type:
 
 ```bash
 gcc --version
 g++ --version
 gdb --version
 ```
 
-If you don't see the expected output or `g++` or `gdb` is not a recognized command, make sure your PATH entry matches the Mingw-w64 binary location where the compiler tools are located.
+You should see output that states which versions of GCC, g++ and GDB you have installed. If this is not the case, make sure your PATH entry matches the Mingw-w64 binary location where the compiler tools are located or reference the [troubleshooting section](/docs/cpp/config-mingw.md#_check-your-mingw-installation).
 
-If the compilers do not exist at that PATH entry, make sure you followed the instructions on the [MSYS2 website](https://www.msys2.org/) to install Mingw-w64.
+## Create a Hello World App
 
-## Hello World
+To make sure the compiler is installed and configured correctly, lets create a Hello World C++ program.
 
-To make sure the compiler is installed and configured correctly, we'll create the simplest Hello World C++ program.
+### Create a C++ file
 
-Create a folder called "HelloWorld" and open VS Code in that folder (`code .` opens VS Code in the current folder):
+1. On Windows, launch a Windows command prompt (Enter **Windows command prompt** in the Windows search bar). On Mac and Linux, you can enter these commands in the terminal.
+1. Run the following commands. They are creating an empty folder called `projects` where you can place all your VS Code projects. The next commands create and navigate you to a subfolder called `helloworld`. From there, you are opening `helloworld` directly in VS Code using the `code` command.
 
-```bash
-mkdir HelloWorld
-cd HelloWorld
+```bat
+mkdir projects
+cd projects
+mkdir helloworld
+cd helloworld
 code .
 ```
 
-The "code ." command opens VS Code in the current working folder, which becomes your "workspace". Accept the [Workspace Trust](/docs/editor/workspace-trust.md) dialog by selecting **Yes, I trust the authors** since this is a folder you created.
+The "code. " command opens VS Code in the current working folder, which becomes your "workspace". Accept the [Workspace Trust](/docs/editor/workspace-trust.md) dialog by selecting **Yes, I trust the authors** since this is a folder you created.
```

Comment:
Need the space between "code" and ".".
This is the command to open VS Code in the current folder "."

Reply:
Fixed, when doing the merge conflict